### PR TITLE
JMLL-1094 Update SRE/Security pages

### PR DESCRIPTION
**Pull request by SamDudley**

### CODE REVIEW

File: leavers/views/security_team.py [hunk 1]
```diff
@@ -486,6 +491,7 @@ def get_kit_info(self) -> List[ServiceInfo]:
                 self.leaving_request, rosa_kit_field
             )
             if rosa_task_log:
+                rosa_info["comment"] = rosa_task_log.notes or ""
```
Comment (jafacakes2011) [inline leavers/views/security_team.py:494]: Note for me to test this locally

**Review by jafacakes2011 — APPROVED** [review]
LGTM, Just checking the notes locally :)

File: leavers/views/security_team.py [hunk 2]
```diff
@@ -496,6 +503,17 @@ def get_kit_info(self) -> List[ServiceInfo]:
...
+            # Get the most recent note referring to this field.
+            most_recent_rosa_task_log: Optional[TaskLog] = (
+                self.leaving_request.task_logs.filter(
+                    reference=f"LeavingRequest.{rosa_kit_field}"
+                )
+                .order_by("-created_at")
+                .first()
+            )
+            if most_recent_rosa_task_log:
+                rosa_info["comment"] = most_recent_rosa_task_log.notes or ""
```
Comment (jafacakes2011) [inline leavers/views/security_team.py:515]: @SamDudley I've updated the logic so that the note comes from the latest task log that references the field

Comment (SamDudley) [reply leavers/views/security_team.py:515]: Looks good to me!

**Pull request merged**